paasta_tools/marathon_serviceinit.py

```diff
@@ -138,20 +138,30 @@ def get_verbose_status_of_marathon_app(app):
     output.append("    App created: %s (%s)" % (str(create_datetime), humanize.naturaltime(create_datetime)))
     output.append("    Tasks:")
 
-    rows = [("Mesos Task ID", "Host deployed to", "Deployed at what localtime")]
+    rows = [("Mesos Task ID", "Host deployed to", "Deployed at what localtime", "Health checks")]
     for task in app.tasks:
         local_deployed_datetime = datetime_from_utc_to_local(task.staged_at)
         if task.host is not None:
             hostname = "%s:%s" % (task.host.split(".")[0], task.ports[0])
         else:
             hostname = "Unknown"
+        health_check_results = [health_check.alive for health_check in task.health_check_results
+                                if health_check.alive is not None]
+        if not health_check_results:
+            health_check_status = PaastaColors.grey("UNKNOWN")
```

Comment:
How about `N/A`? (as not every service has healthchecks and unknown sounds like something might be wrong